test/Generator.V3.Tests/Generator.V3.Tests.csproj

```diff
@@ -0,0 +1,101 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
This whole ItemGroup block shouldn't be needed.

Reply:
```
<ItemGroup>
    <Content Include="TestDtdlModels\*.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
  ```
  
  You'll want this just like it's done in the V2 project

Reply:
fixed.